src/cascadia/TerminalSettingsEditor/ProfileViewModel.cpp

```diff
@@ -76,6 +76,10 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
             {
                 _NotifyChanges(L"HideIcon");
             }
+            else if (viewModelProperty == L"Padding")
+            {
+                _NotifyChanges(L"LeftPadding", L"TopPadding", L"RightPadding", L"BottomPadding");

```

Comment:
In the directional padding setters `Padding(padding)` will call `_notifyChanges(L "Padding")` and `_notifyChanges(L "HasPadding")`. If I call `_notifyChanges` for all directional paddings here, it works, including the `ClearPadding` case.